server/handler/pull_request_review.go

```diff
@@ -39,13 +40,55 @@ func (h *PullRequestReview) Handle(ctx context.Context, eventType, deliveryID st
 		return errors.Wrap(err, "failed to parse pull request review event payload")
 	}
 
+	pr := event.GetPullRequest()
+	repo := event.GetRepo()
+	owner := repo.GetOwner().GetLogin()
+	number := event.GetPullRequest().GetNumber()
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
+
+	client, err := h.NewInstallationClient(installationID)
+	if err != nil {
+		return err
+	}
+
+	v4client, err := h.NewInstallationV4Client(installationID)
+	if err != nil {
+		return err
+	}
+
 	ctx, _ = h.PreparePRContext(ctx, installationID, event.GetPullRequest())
 
+	mbrCtx := NewCrossOrgMembershipContext(ctx, client, owner, h.Installations, h.ClientCreator)
+	prctx, err := pull.NewGitHubContext(ctx, mbrCtx, client, v4client, pull.Locator{
+		Owner:  owner,
+		Repo:   repo.GetName(),
+		Number: number,
+		Value:  pr,
+	})
+	if err != nil {
+		return err
+	}
+
+	fc := h.ConfigFetcher.ConfigForPR(ctx, prctx, client)
+
+	if !h.affectsApproval(event.GetReview().GetState(), fc.Config.ApprovalRules) {
+		return nil
+	}
+
 	return h.Evaluate(ctx, installationID, common.TriggerReview, pull.Locator{
```

Comment:
Sorry about all the boilerplate you have to copy to get access to the config prior to calling `Evaluate`. But now that we have a `FetchedConfig`, this should probably call `ValidateFetchedConfig` and `EvaluateFetchedConfig` instead, since `Evaluate` repeats the client creation and config fetching.